backend/src/packages/articles/article.repository.ts

```diff
@@ -61,17 +69,25 @@ class ArticleRepository implements IArticleRepository {
     genreId,
     titleFilter,
     authorId,
+    showFavourites,
+    requestUserId,
   }: {
     userId?: number;
     hasPublishedOnly?: boolean;
+    requestUserId: number;
```

Comment:
But `requestUserId` and `userId` are the same ID for current user

Reply:
Yes, but one is for filters and optional and second for isFavourite status and should always be accessible

Reply:
As in `getIsFavouriteSubQuery` you're waiting for userId as number | null, then I believe we can use existing userId